packages/databyss-editor/package.json

```diff
@@ -36,7 +36,7 @@
     "is-hotkey": "^0.1.5",
     "lodash": "^4.17.15",
     "lorem-ipsum": "^2.0.3",
-    "slate-react": "0.57.2",
+    "slate-react": "0.58.4",
```

Comment:
this change causes the cursor to jump, an issue we fixed with our fork of `slate`